src/main/java/org/aszjch/demoapp/infrastructure/storage/TempFilesService.java

```diff
@@ -0,0 +1,43 @@
+package org.aszjch.demoapp.infrastructure.storage;
+
+import jakarta.annotation.PostConstruct;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.aszjch.demoapp.infrastructure.config.TempFilesProperties;
+import org.springframework.stereotype.Component;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+
+@Slf4j
+@Component
+@RequiredArgsConstructor
+public class TempFilesService {
```

Comment:
Czy to Serwajs?

Reply:
W sumie to nie, jakaś pokraka z tego wyszła trochę. Niby converter by pasował patrząc po nazwie pliku, ale znowu tak nie do końca, skoro on przegrywa treść z multiparta na zwykły file...